velox/common/memory/SharedArbitrator.cpp

```diff
@@ -427,6 +427,8 @@ bool SharedArbitrator::arbitrateMemory(
 
   VELOX_CHECK_LT(freedBytes, growTarget);
   RECORD_METRIC_VALUE(kMetricArbitratorGlobalArbitrationCount);
+  addThreadLocalRuntimeStat(
```

Comment:
Can you define this as consts in SharedArbitrator.h? See for kSpillFillTime in Operator.h for example. Thanks!